### PR TITLE
Small ecstatic fix/update

### PR DESCRIPTION
**Pull request by leangseu-edx**
- update submission navigation styling
- update review action styling
- make container more responsive

### CODE REVIEW

File: src/components/StatusBadge.jsx [hunk 1]
```diff
@@ -13,17 +13,21 @@ export const StatusBadge = ({ className, status }) => {
...
+      args = { label: 'Grading in progress', variant: 'light' };
+      break;
+    case statuses.graded:
+      args = { label: 'Grading Complete', variant: 'success' };
+      break;
+    case statuses.locked:
+      args = { label: 'Locked by you', variant: 'warning' };
+      break;
+    default:
+      break;
```
Comment (muselesscreator) [inline src/components/StatusBadge.jsx]: why?

Comment (leangseu-edx) [reply src/components/StatusBadge.jsx]: Earlier it was wrong on in progress and locked. It was confusing when I did filtering. Since I edit it, might as well optimize it a little. I can change it to `else if` if that is prefer.

File: src/containers/ListView/index.jsx [hunk 2]
```diff
@@ -92,6 +88,26 @@ export class ListView extends React.Component {
...
+                },
+                {
+                  name: 'Graded',
+                  value: 'graded',
+                },
+                {
+                  name: 'In Progress',
+                  value: 'in-progress'
+                },
+              ]
```
Comment (muselesscreator) [inline src/containers/ListView/index.jsx]: This was for a separate ticket, and should probably be in its own PR

Comment (leangseu-edx) [reply src/containers/ListView/index.jsx]: It is not polish to be honest. It was more of a discovery. I just wanted to get ride of the change was in my stash 😅 . I don't mind remove it for now.

File: src/containers/ReviewModal/ReviewActions.jsx [hunk 3]
```diff
@@ -36,7 +35,7 @@ export const ReviewActions = ({
         <SubmissionNavigation />
       </div>
     </ActionRow>
-  </div>
+    </div>
```
Comment (muselesscreator) [inline src/containers/ReviewModal/ReviewActions.jsx]: This is separate from the rubric container fix.
Also, why the added indentation?

File: src/components/StatusBadge.jsx (hunk 1 above)
Comment (muselesscreator) [reply src/components/StatusBadge.jsx]: If anything, I would set up a dict for this lookup
```
const statusArgs = {
  <statuses.ungraded>: { label: 'Ungraded', variant: 'primary' },
  ...
};
...
args = statusArgs[status]
```

File: src/containers/ReviewModal/ReviewActions.jsx (hunk 3 above)
Comment (leangseu-edx) [reply src/containers/ReviewModal/ReviewActions.jsx]: This commit has nothing to do with rubric container at all. It is mostly just updating loose end from the merge.

**Review by muselesscreator — CHANGES_REQUESTED** [review]
So... the code in this is generally good, but the PR feels like 3 separate things...
 * A fix to the action-row responsive behavior
 * An partial implementation of the DataTable filters
 * An update to Rubric container responsiveness.
 
 Please try to separate out these concerns.
 
 The Rubric responsiveness seems better.
 The fixes to things from merge clobber are reasonable, but should be separate.
 The DataTable filter stuff should hold off until that ticket is ready in its entirety.

[Edit] if the Rubric Responsiveness is included in the merge-clobber resolution, that is reasonable, but unclear.

File: src/containers/ReviewModal/ReviewActions.jsx (hunk 3 above)
Comment (muselesscreator) [reply src/containers/ReviewModal/ReviewActions.jsx]: first concern resolved.  Still an extra indentation added :-p

**Review by muselesscreator — APPROVED** [review]
LGTM
